goserver/session/session.go

```diff
@@ -0,0 +1,114 @@
+package session
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/base64"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
+
+var (
+	// SessionCookieSecret is used to encrypt the cookie session id.
+	SessionCookieSecret = ""
```

Comment:
Ya this is better, but ideally we also wouldn't use global variables to configure packages since then multiple users of this package couldn't use different configuration values. Not a big deal at the moment, but if you decide to consolidate all logic under SessionManager it might make more sense to move this configuration to a field on SessionManager.

Reply:
Done.